community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md

```diff
@@ -50,6 +50,32 @@ This creates a controller node with the following attributes:
 For a complete example using this module, see
 [slurm-gcp-v5-cluster.yaml](../../../examples/slurm-gcp-v5-cluster.yaml).
 
+### Reconfigure Option
+The schedmd-slurm-gcp-v5-controller module supports the reconfiguration of
+partitions and slurm configuration in a running, active cluster. This option is
+activated through the `enable_reconfigure` setting:
+
+```yaml
+- id: slurm_controller
+  source: community/modules/scheduler/schedmd-slurm-gcp-v5-controller
+  settings:
+    enable_reconfigure: true
+```
+
+Reconfigure has some additional requirements:
+
+* The Pub/Sub API must be activated in the target project:
+  `gcloud services enable file.googleapis.com --project "<<PROJECT_ID>>"`
+* Python package dependencies need to be installed with pip in the environment
```

Comment:
Python must be installed ([Python](https://github.com/SchedMD/slurm-gcp/blob/3979e81fc5e4f021b5533a23baa474490f4f3614/docs/glossary.md#python) is installed.
Required Version: >= 3.6.0, < 4.0.0) and python packages dependencies ...

It is pretty hard impossible to follow these instructions as they are there, since it will say: 
pip3 install -r ../../../scripts/requirements.txt --user

But that file is not in our repo, and so they have to find its contents in the slurm on gcp repo, and now that file contains 63 lines of dependencies, which we suspect are not all needed for reconfigure.. I started a thread with SchedMD to see if we could narrow down the dependencies and, at least, point them to the right file here.

Reply:
I clarified the python dependency in the warning of this module. I also added `enable_reconfigure` as one of the triggering options for the requirement. Rather than rewrite them here, I added a link to the description and inform the user to learn more there.